robotoff/prediction/ocr/packager_code.py

```diff
@@ -34,75 +34,136 @@ def process_fsc_match(match) -> str:
     return "FSC-{}".format(fsc_code).upper()
 
 
-PACKAGER_CODE: Dict[str, OCRRegex] = {
-    "fr_emb": OCRRegex(
-        re.compile(r"emb ?(\d ?\d ?\d ?\d ?\d) ?([a-z])?(?![a-z0-9])"),
-        field=OCRField.text_annotations,
-        lowercase=True,
-        processing_func=process_fr_emb_match,
-    ),
-    "fsc": OCRRegex(
-        re.compile(r"fsc.? ?(c\d{6})"),
-        field=OCRField.text_annotations,
-        lowercase=True,
-        processing_func=process_fsc_match,
-    ),
-    "eu_fr": OCRRegex(
-        re.compile(
-            r"fr (\d{2,3}|2[ab])[\-\s.](\d{3})[\-\s.](\d{3}) (ce|ec)(?![a-z0-9])"
+def process_USDA_match_to_flashtext(match) -> Optional[Any]:
```

Comment:
It's a bit lazy to use Any here ;-)
Isn't it Optional[str] ?

Reply:
Ok, I wasn't sure. I must admit that I am not really used to types. Thanks !